packages/env/lib/cli.js

```diff
@@ -109,6 +109,28 @@ module.exports = function cli() {
 		},
 		withSpinner( env.clean )
 	);
+	yargs.command(
+		'log',
+		'displays PHP and Docker logs for given WordPress environment.',
+		( args ) => {
+			args.positional( 'environment', {
+				type: 'string',
+				describe: 'Which environment to display the logs from.',
+				choices: [ 'development', 'tests' ],
+				default: 'development',
+			} );
+			args.positional( 'watch', {
+				type: 'boolean',
+				describe: 'If true, output logs to the console as they happen.',
+				default: true,
+			} );
+		},
+		withSpinner( env.log )
+	);
+	yargs.example(
+		'$0 log --watch=false --environment=tests',
```

Comment:
If `environment` is a positional and `watch` is an option, this should be:

```
'$0 log tests --no-watch'
```

Reply:
done!